src/main/scala/epic/logo/ArgmaxInferencer.scala

```diff
@@ -0,0 +1,7 @@
+package epic.logo
+
+trait ArgmaxInferencer[T, Y, W] {
+
+  def argmax(weights : Weights[W], instance : T) : (Y, FeatureVector[W], Double)
```

Comment:
Well that certainly sounds like some really interesting work you are all doing! It also sounds like I should buy you a cup of coffee and ask you many questions, most of which you'll just have to smile, nod, and say "well you didn't sign an NDA so..." =D